### ISSUE TITLE
add support for incoming links in direct messages

### ISSUE THREAD

**Issue by realies** [EionRobb/purple-instagram#2]
at the moment incoming messages with links do not seem to be handled

**Comment by dennisnez** [comment]
Yes please!  It's extremely frustrating when someone sends me a link ... sometimes just a link ... and I have  no idea that a message was sent to me :'(
